tables/he-common-vowels-ihbc.uti

```diff
@@ -22,476 +22,94 @@
-noback always צֱּ 5-2346-26  # tsadi with dagesh and hataf segol
-noback always צֲּ 5-2346-25  # tsadi with dagesh and hataf patah
-noback always צֳּ 5-1235-2346  # resh with dagesh and hataf qamats
-noback always צִּ 5-2346-24  # Tsadi with dagesh and hiriq
-noback always צֵּ 5-2346-34  # tsadi with dagesh and tsere
-noback always צֶּ 5-2346-15  # tsadi with dagesh and segol
-noback always צַּ 5-2346-14  # tsadi with dagesh and patah
-noback always צָּ 5-2346-126  # tsadi with dagesh and qamats
-noback always קְּ 5-12345-3  # qof with dagesh sheva
-noback always קֱּ 5-12345-26  # qof dagesh  hataf segol
-noback always קֲּ 5-12345-25  # qof dagesh  hataf patah
-noback always קֳּ 5-12345-345  # qof dagesh  hataf qamats
-noback always קִּ 5-12345-24  # qof  with dagesh and hiriq
-noback always קֵּ 5-12345-34  # qof  with dagesh and tsere
-noback always קֶּ 5-12345-15  # qof  with dagesh and segol
-noback always קַּ 5-12345-14  # qof  with dagesh and patah
-noback always קָּ 5-12345-126  # qof dagesh  qamats
-noback always רְּ 5-1235-3  # Resh with dagesh and sheva
-noback always רֱּ 5-1235-26  # resh with dagesh and hataf segol
-noback always רֲּ 5-1235-25  # resh with dagesh and hataf patah
-noback always רֳּ 5-1235-345  # resh with dagesh and hataf qamats
-noback always רִּ 5-1235-24  # Resh with dagesh and hiriq
-noback always רֵּ 5-1235-34  # Resh with dagesh and tsere
-noback always רֶּ 5-1235-15  # Resh with dagesh segol
-noback always רַּ 5-1235-14  # resh with dagesh and patah
-noback always רָּ 5-1235-126  # Resh with dagesh and qamats
-noback always שְּׁ 5-146-3  #  Shin with dot dagesh and sheva
-noback always שְּׂ 5-156-3  #  Sin with dagesh and sheva
-noback always שֱּׁ 5-146-26  # shin  hataf segol
-noback always שֱּׂ 5-156-26  # sin  hataf segol
-noback always שֲּׁ 5-146-25  # shin  hataf patah
-noback always שֲּׂ 5-156-25  # sin  hataf patah
-noback always שֳּׁ 5-146-345  # shin  hataf qamats
-noback always שֳּׂ 5-156-345  # sin  hataf qamats
-noback always שִּׁ 5-146-24  # shin with hiriq
-noback always שִּׂ 5-156-24  # sin with hiriq
-noback always שֵּׁ 5-146-34  # shin with tsere
-noback always שֵּׂ 5-156-34  # sin with tsere
-noback always שֶּׁ 5-146-15  # shin with segol
-noback always שֶּׂ 5-156-15  # sin with segol
-noback always שַּׁ 5-146-14  # shin with patah
-noback always שַּׂ 5-156-14  # sin with patah
-noback always שָּׁ 5-146-126  # shin  qamats
-noback always שָּׂ 5-156-126  # sin  qamats
-noback always שְּׁ 5-146-3  # Shin with dot dagesh and sheva
-noback always שְּׂ 5-156-3  #  Sin with dagesh and sheva
-noback always כְּ 13-3  # KAF with dagesh  sheva
-noback always תְּ 1256-3  # tav with daggesh sheva
+# Cantillation marks included in IHBC
+sign \x0591 23  # Atnakh/Etnahta
+sign \x0594 2  # Zaqef Qatan
 
-include he-common-consonants.uti
+# Define vowel class (also includes Etnahta and zaqef qatan)
+attribute vowel /x0591/x0594\x05b0\x05b1\x05b2\x05b3\x05b4\x05b5\x05b6\x05b7\x05b8\x05b9\x05ba\x05bb\x05c7
```

Comment:
I just discovered that this contains slashes where I think they should be backslashes

```suggestion
attribute vowel \x0591\x0594\x05b0\x05b1\x05b2\x05b3\x05b4\x05b5\x05b6\x05b7\x05b8\x05b9\x05ba\x05bb\x05c7
```